### PR TITLE
Add Kubernetes Assets

### PR DESCRIPTION
**Pull request by cmrust**
Would like to have the Kubernetes assets for this work, located locally, within this repo for quicker development turn-around.

This is all tested and working. Please provide your thoughts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,24 @@ If you want to change any of these parameters simply add them to a match stanza.
 </match>
 ```
 
+## Kubernetes
```
Comment (alexouzounis) [inline README.md]: Could you also add a section and a badge from Quay to provide a link to the registry like so:

[Docker Repository on Quay](https://quay.io/repository/solarwinds/fluentd-kubernetes)

File: README.md [hunk 2]
```diff
@@ -74,6 +74,24 @@ If you want to change any of these parameters simply add them to a match stanza.
...
+```
+kubectl apply -f kubernetes/fluentd-daemonset-papertrail.yaml
+```
+
+The Dockerfile that generates the image used in this DaemonSet, can be found at `docker/Dockerfile`. Changes can be pushed to the official repo by running:
+
+```
+cd docker
+docker build -t quay.io/solarwinds/fluentd-kubernetes:v0.12.33-debian-papertrail .
+docker push quay.io/solarwinds/fluentd-kubernetes:v0.12.33-debian-papertrail
```
Comment (alexouzounis) [inline README.md]: Might be worth adding a `Makefile` for the build / push process of the docker image

File: README.md (hunk 1 above)
Comment (cmrust) [reply README.md]: Yes, will add this.

File: README.md (hunk 2 above)
Comment (cmrust) [reply README.md]: Going to remove these altogether and setup a build trigger from the Quay.io side.

**Comment by cmrust** [comment]
Hey @alexouzounis, was having some issues with the Quay.io build trigger. Will bring it up on Friday. In the meantime I've added the status badges and release functionality to the Makefile. Can you please approve again?

**Pull request merged**